packages/syntax-core/src/SelectList/SelectList.module.css

```diff
@@ -0,0 +1,80 @@
+.selectContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.opacityOverlay {
+  opacity: 0.3;
+}
+
+.outerTextContainer {
+  padding-inline-start: 4px;
```

Comment:
@nathan-ong should we also add padding to the end? That way we'll be using symmetrical padding

```jsx
padding-inline-end: 4px;
padding-inline-start: 4px;
```